profile/README.md

```diff
@@ -0,0 +1,16 @@
+![Adoptium Banner](/images/banner.png)
```

Comment:
We should avoid to use the magenta/red color for text on the dark blue background since it can be really hard to read for some persons. A first check shows that it is against https://www.w3.org/WAI/WCAG2AAA-Conformance
@gdams can you use white text?